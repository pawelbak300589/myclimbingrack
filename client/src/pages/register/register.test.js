import React from 'react';
import { shallow } from "enzyme";

import RegisterPage from "./register.component";

it('should render RegisterPage component', () => {
    expect(shallow(<RegisterPage />)).toMatchSnapshot();
});