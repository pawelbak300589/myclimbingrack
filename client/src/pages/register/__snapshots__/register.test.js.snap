// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render RegisterPage component 1`] = `ShallowWrapper {}`;
